src/Bridges/indicatorbridge.jl

```diff
@@ -36,13 +36,12 @@ function bridge_constraint(::Type{<:IndicatorActiveOnFalseBridge}, model::MOI.Mo
         f.constants
     )
     ci = MOI.add_constraint(model, f2, MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}(s.set))
-
-    return IndicatorActiveOnFalseBridge{T, MOI.VectorAffineFunction{T}, S}(z2, zo_cons, dcons, ci)
+    return IndicatorActiveOnFalseBridge{T,F,S}(z2, zo_cons, dcons, ci)
 end
 
-function MOI.supports_constraint(::Type{<:IndicatorActiveOnFalseBridge},
-                                 ::Type{<:MOI.VectorAffineFunction},
-                                 ::Type{<:MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO}})
+function MOI.supports_constraint(::Type{<:IndicatorActiveOnFalseBridge{T}},
+                                 ::Type{F},
```

Comment:
Why not just do `Type{<:MOI.VectorAffineFunction}` here ? `F` is not used

Reply:
Fixed